src/engine/blocks.js

```diff
@@ -506,6 +511,16 @@ class Blocks {
         this._cache._monitored = null;
     }
 
+    /**
+     * Emit a project changed event if this is a block container
+     * that can affect the project state.
+     */
+    emitProjectChanged () {
+        if (!this.forceNoGlow) {
```

Comment:
I'm wondering if we want to take this time to rename that option, or create a separate, explicit option related to project state here?